docs/get-started.md

```diff
@@ -29,31 +29,62 @@ title: Get started
 {% band header="Include PatternFly Elements JavaScript" %}
   There are a few options for including PatternFly Elements into your site or app.
 
-  ### In HTML
-  You may load the JavaScript via `<script type="module">`. List out all the 
-  components you may include on your page, and the browser will fetch the 
-  dependencies dynamically on load.
-  [Learn more about JavaScript modules][modules].
-
-  In this example, we load the [card](/components/card/) modules from 
-  [JSPM](https://jspm.dev).
-
-  ```html
-  <script type="module" src="https://jspm.dev/@patternfly/elements/pf-card/pf-card.js"></script>
-  ```
-
-  <a id="in-an-app"></a>
-
   ### In JavaScript modules
 
   Include the PatternFly Element web component and its dependencies within the app.
   When using a bundler such as [esbuild](https://esbuild.github.io/) or [rollup](https://rollupjs.org)
   with [@rollup/plugin-node-resolve](https://www.npmjs.com/package/@rollup/plugin-node-resolve), use
-  [bare module specifiers](https://lit.dev/docs/tools/requirements/) to import the components.
+  [bare module specifiers](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#importing_modules_as_bare_names) to import the components.
 
   ```javascript
   import '@patternfly/elements/pf-card/pf-card.js';
   ```
+
+  ### In HTML
+  You may load the components in HTML using script tags via `<script type="importmap">` and `<script type="module">`. 
+  
+  List out all the components you want to use on your page or app, and use [generator.jspm.io](https://generator.jspm.io/#U2VhYGAIzSvJLMlJTWEAACKwkqAOAA) to generate a JSON importmap for the component entry points.  [Learn more about importmaps](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap). 
```

Comment:
```suggestion
  List all the components you want to use on your page or app, and use [generator.jspm.io](https://generator.jspm.io/#U2VhYGAIzSvJLMlJTWEAACKwkqAOAA) to generate a JSON importmap for the component entry points.  [Learn more about importmaps](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap). 
```

"Use generator" is a little vague, I think we need a whole section on how to manually write the map or how to use the generator

Reply:
Ok, I'll try and stub something out there.